app/operator/konnect/control-plane-watch-namespaces.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Limiting namespaces watched by ControlPlane"
+description: "Learn how to limit the namespaces that `ControlPlane` watches."
```

Comment:
```suggestion
description: "Learn how to limit the namespaces that ControlPlane watches."
```
Don't want to use backticks in the description, it won't display right in search results.